--- cadaver-0.22.2.orig/libneon/ne_request.c
+++ cadaver-0.22.2/libneon/ne_request.c
@@ -176,7 +176,7 @@
 
 /* Returns hash value for header 'name', converting it to lower-case
  * in-place. */
-static inline unsigned int hash_and_lower(char *name)
+static unsigned int hash_and_lower(char *name)
 {
     char *pnt;
     unsigned int hash = 0;
@@ -868,7 +868,7 @@
 /* remove trailing EOL from 'buf', where strlen(buf) == *len.  *len is
  * adjusted in accordance with any changes made to the string to
  * remain equal to strlen(buf). */
-static inline void strip_eol(char *buf, ssize_t *len)
+static void strip_eol(char *buf, ssize_t *len)
 {
     char *pnt = &buf[*len-1];
     while (pnt >= buf && (*pnt == '\r' || *pnt == '\n')) {
