--- cadaver-0.22.2.orig/macros/neon.m4
+++ cadaver-0.22.2/macros/neon.m4
@@ -829,7 +829,7 @@
 if test "$enable_warnings" = "yes"; then
    case $GCC:`uname` in
    yes:*)
-      CFLAGS="$CFLAGS -Wall -ansi-pedantic -Wmissing-declarations -Winline -Wshadow -Wreturn-type -Wsign-compare -Wundef -Wpointer-arith -Wcast-align -Wbad-function-cast -Wimplicit-prototypes -Wformat-security"
+      CFLAGS="$CFLAGS -Wall -Wmissing-declarations -Winline -Wshadow -Wreturn-type -Wsign-compare -Wundef -Wpointer-arith -Wcast-align -Wbad-function-cast -Wformat-security"
       if test -z "$with_ssl" -o "$with_ssl" = "no"; then
 	 # OpenSSL headers fail strict prototypes checks
 	 CFLAGS="$CFLAGS -Wstrict-prototypes"
